docs/architecture/cloud-native/service-mesh-communication-infrastructure.md

```diff
@@ -59,14 +59,20 @@ We next move to distributed data in cloud-native systems and the benefits and ch
 
 - [About Ingress in Azure Kubernetes Service (AKS)](https://vincentlauzon.com/2018/10/10/about-ingress-in-azure-kubernetes-service-aks/)
 
-- [Practical gRPC](https://www.worldcat.org/title/practical-grpc/oclc/1042342319)
+- [Practical gRPC](https://www.amazon.com/Practical-gRPC-Joshua-Humphries-ebook/dp/B07DMS44WT/ref=sr_1_1?keywords=Practical+gRPC&qid=1581887357&sr=8-1)
```

Comment:
This link has search keywords. Also, I am not sure if we need to link this book? The gRPC official docs are great.

Reply:
As discussed, removed link to this external book